raft/src/test/java/org/apache/kafka/raft/CandidateStateTest.java

```diff
@@ -58,193 +60,229 @@ private CandidateState newCandidateState(VoterSet voters) {
         );
     }
 
-    @Test
-    public void testSingleNodeQuorum() {
-        CandidateState state = newCandidateState(voterSetWithLocal(IntStream.empty()));
+    @ParameterizedTest
+    @ValueSource(booleans = { true, false })
```

Comment:
I don't believe adding whitespace around braces like this is allowed by checkstyle (I guess you will find out)

Reply:
Looks like Kafka's checkstyle accepts both (`{ true, false }` and `{true, false}`). If we want to change this, do you mind if we do this in another PR? I use this style in a lot of places.

Reply:
I'm ok with either way. I'm just surprised checkstyle accepts it.